packages/node-build-scripts/generate-icons-source

```diff
@@ -36,21 +36,21 @@ writeLinesToFile(
     "_icon-map.scss",
     '@import "icon-variables";',
     "$icons: (",
-    ...ICONS_METADATA.map(icon => `  "${toShortName(icon)}": $${icon.className},`),
+    ...ICONS_METADATA.map(icon => `  "${icon.iconName}": $${toLongName(icon)},`),
     ");",
 );
 
 // simple variable definitions
-writeLinesToFile("_icon-variables.scss", ...ICONS_METADATA.map(icon => `$${icon.className}: "${icon.content}";`));
+writeLinesToFile("_icon-variables.scss", ...ICONS_METADATA.map(icon => `$${toLongName(icon)}: "${icon.content}";`));
 
 // map ENUM_NAME to pt-icon-class-name
-writeLinesToFile("iconClasses.ts", ...exportIconConsts(icon => icon.className));
+writeLinesToFile("iconClasses.ts", ...exportIconConsts(toLongName));
```

Comment:
@adidahiya thoughts on deleting this file as it's simply `Classes.iconClass(IconNames.AIRPLANE)`?

Reply:
hm yeah I guess that's fine... can you add it to the migration script?